css/style.css

```diff
@@ -42,6 +48,14 @@ nav {
   text-decoration: none;
 }
 
+.mobile-nav-list li {
+  text-align: left;
+  margin-top: 20px;
+  border-bottom: 2px solid #6f6c6b;
+  max-width: 100%;
+  padding: 15px;
+}
```

Comment:
- [ ] Your mobile hamburger button disappears from the page in the mobile view for any view width larger than 250px like the images below so please fix that.

![image](https://user-images.githubusercontent.com/68796232/161760562-9cfe1229-a5ba-4f52-a89e-87a6022ff7e2.png)  ![image](https://user-images.githubusercontent.com/68796232/161760519-e49d5da1-67e5-42fc-a7e6-b72b93596e01.png)

Reply:
@Omar-Muhamad 
This is how it appears on my end, i do not know why its different in yours.
please how do i fix this.

![image](https://user-images.githubusercontent.com/97225349/161766195-90040f73-33a5-44af-be43-aa9b2fa309c5.png)

Reply:
@Omar-Muhamad these are different sizes
![image](https://user-images.githubusercontent.com/97225349/161767057-fcbeca0f-60e5-48b4-8e06-3cf841e70d1f.png)
![image](https://user-images.githubusercontent.com/97225349/161767100-a27f2947-0bf2-4eda-9355-e50f01816b0b.png)